### PR TITLE
docs for system table integration

### LINKED ISSUE #1127: [DOC] - Add document for system tables integration
Add documentation for system tables integration.

### PR DESCRIPTION
**Pull request by aman-db**
closes #1127 
This PR contains docs for System Table Integration

### CODE REVIEW

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: "System Table Configuration"
+date: 2024-01-23T16:30:21+05:30
+---
+
+## Configuration changes required for System Table Integration
+
+There is no unique configuration process for System Table ; however, the format for the **BELOW CONFIGURATIONS**
```
Comment (gueniai) [inline docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md:8]: This sentence is a bit hard to parse. I think we could do something close to: "The only change required to the configuration is the following: auditlogprefix_source_path - Instead of adding a fully qualified path (s3 or GC) for auditlog, add keyword system in this column. This will enable the system table integration. "

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+are specific for System Table Integration. For all other configurations, please follow the
+[Configuration]({{%relref "DeployOverwatch/ConfigureOverwatch/Configuration.md"%}})
+
+* **auditlogprefix_source_path** - Instead of adding a fully qualified path (s3 or GC) for auditlog, 
+add keyword **system** in this column. This will enable the system table integration.
+
+**Same congifuration changes can be used for migrating from legacy to system table integration.**
```
Comment (gueniai) [inline docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md:15]: If customers are upgrading to 0800 they can't use legacy.

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+{{% notice note %}}
+For Azure deployments since Eventhub is not required for System Table Integration, we can leave these 
+fields blank - eh_name, Event hub name, eh_name, eh_conn_string, aad_tenant_id, aad_client_id, aad_client_secret_key, 
+aad_authority_endpoint
+{{% /notice %}}
+
+## Configuration changes required for Multi Account System Table Integration
+The process for achieving multi-account deployment is straightforward. In the workspace of each additional account, 
+a new sql_endpoint/warehouse needs to be created. The endpoint details are then passed to Overwatch through the configuration,
```
Comment (gueniai) [inline docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md:25]: we don't actually need customers to create a new endpoint just for this. We can say we require a sql endpoint, and encourage them to use serverless since that would be the best approach

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+**Same congifuration changes can be used for migrating from legacy to system table integration.**
+
+{{% notice note %}}
+For Azure deployments since Eventhub is not required for System Table Integration, we can leave these 
+fields blank - eh_name, Event hub name, eh_name, eh_conn_string, aad_tenant_id, aad_client_id, aad_client_secret_key, 
+aad_authority_endpoint
+{{% /notice %}}
+
+## Configuration changes required for Multi Account System Table Integration
+The process for achieving multi-account deployment is straightforward. In the workspace of each additional account,
```
Comment (gueniai) [inline docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md:24]: Instead of "In the workspace of each additional account" reword to something similar to: "In the configuration table, for each workspace in a different account, you'll need to enter a value for the sql_endpoint column"

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTablePreReq.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+---
+title: "System Table Pre-Requisites"
+date: 2024-01-23T16:27:22+05:30
+---
+
+Once all System Table Integration Pre-requisites are completed, please continue to [Deploy Overwatch]({{%relref "DeployOverwatch/"%}})
```
Comment (gueniai) [inline docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTablePreReq.md:6]: I'd put this sentence at the very end

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTablePreReq.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+section.
+
+This section will walk you through the steps necessary as a prerequisite for System Table Integration with Overwatch.
+
+* **Workspace** should be **UC enabled** and **System Table enabled**. Enabling system tables is pretty easy,
```
Comment (gueniai) [inline docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTablePreReq.md:11]: "Enabling system tables is pretty easy" -> "Enabling system tables is straightforward"

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTablePreReq.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+This section will walk you through the steps necessary as a prerequisite for System Table Integration with Overwatch.
+
+* **Workspace** should be **UC enabled** and **System Table enabled**. Enabling system tables is pretty easy, 
+you do need to have at least one unity-catalog enabled workspace. System tables must be enabled by an account admin. 
+You can enable system tables using the Unity Catalog REST API.  Once enabled, the tables will appear in a catalog 
+called system, which is included in every Unity Catalog metastore. In the system catalog you’ll see schemas such as 
+access and billing that contain the system tables. Follow the documents to enable 
+[System Tables](https://docs.databricks.com/en/administration-guide/system-tables/index.html#enable-system-tables)
+* Overwatch Pipeline **Cluster** must be **UC enabled** (single user and runtime version > 11.3+).
```
Comment (gueniai) [inline docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTablePreReq.md:17]: Say that if multi account they need >13

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTablePreReq.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+* Overwatch latest version(0.7.3.0+) should be deployed in the workspace
```
Comment (gueniai) [inline docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTablePreReq.md:18]: latest version 0.8.0.0

**Review by gueniai — CHANGES_REQUESTED** [review]

**Comment by gueniai** [comment]
We also need to provide a new sample config .xlsx and .csv  in "Configure Overwatch" -> "Configuration"

**Comment by gueniai** [comment]
We need to add a note on the Configure Overwatch -> Validation page about validations performed for System tables

File: docs-compose/overwatch_docs/content/DeployOverwatch/ConfigureOverwatch/Configuration.md [hunk 9]
```diff
@@ -42,39 +42,40 @@ will upgrade the script for you.
...
+| secret_key_dbpat	          | String  | True                        | This will contain the PAT token of the workspace. The key should be present in the secret_scope and should start with dapi.                                                                                                                                                                                                                          |
+| auditlogprefix_source_path | String  | True (only for **AWS/GCP**) | Location of auditlog (**AWS/GCP Only**). The contents under this directory must have the folders with the date partitions like date=2022-12-0 . Use keyword - **system** to fetch data from System Tables (system.access.audit) [System Table Configuration Details]({{%relref "DeployOverwatch/SystemTableIntegration/SystemTableConfiguration"%}}) |
```
[the remark this answers is not in the document]
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/ConfigureOverwatch/Configuration.md:58]: Done

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
+---
+title: "System Table Integration"
+date: 2024-01-22T19:35:23+05:30
+---
+
+{{% notice note %}}
```
[the remark this answers is not in the document]
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md:6]: In my opinion we should put these notice in the public Docs even though it is not overwatch issue. Let me know your thoughts

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+[Databricks Docs for System Tables](https://docs.databricks.com/en/administration-guide/system-tables/index.html)
+
+In our continuous efforts to enhance the efficiency and robustness of Overwatch, we have transitioned to a 
+system table integration for audit logs. This strategic move brings forth numerous advantages, further optimizing the 
+monitoring and logging capabilities of the tool. Currently, our integration focuses on leveraging system table 
+audit logs, but in future releases, we will be expanding our integration to include other system tables as well. Follow 
+these documents to know more about 
+[System Table Audit Logs](https://docs.databricks.com/en/administration-guide/system-tables/audit-logs.html)
+
+Pros of Moving to System Table Integrations:
```
[the remark this answers is not in the document]
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md:24]: done

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+System table integration seamlessly aligns with Overwatch.
+The synergy between system tables and Overwatch ensures a harmonious integration, providing a unified platform for 
+comprehensive monitoring and analysis.
+
+* **Simplified Setup within Overwatch**:
+Integration with system tables leads to a simplified setup within Overwatch. The need to configure audit log settings 
+independently within the Overwatch configurations is eliminated. This simplicity ensures a more user-friendly experience
+for both setup and ongoing management. 
+
+* **Reduced Dependency on IAM Roles and Service Accounts**:
```
[the remark this answers is not in the document]
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md:36]: We have mentioned this will be specifically for Audit Log management. If you want we can rename the heading to 
`Reduced Dependency on IAM Roles and Service Accounts for Audit Logs setup`

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+The move to system table integrations diminishes the necessity for setting up extra IAM roles or service accounts 
+specifically for audit log management within Overwatch. This reduction in dependencies streamlines the integration 
+process, enhancing the overall efficiency of the tool.
+
+* **Effortless Migration**:
+The migration process to system tables is designed to be effortless within the Overwatch framework. 
+The transition is smooth, minimizing disruptions and ensuring continuous monitoring capabilities 
+without requiring extensive reconfigurations.
+
+* **Extended Data Retention for Azure Deployments in Overwatch**:
```
[the remark this answers is not in the document]
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md:46]: done

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md [hunk 14]
```diff
@@ -0,0 +1,76 @@
...
+Particularly advantageous for Azure deployments within the Overwatch environment, the use of system tables extends the 
+audit log retention period beyond 30 days for audit log bronze. This eliminates the previous dependency on Events Hub, 
+which had periodic data flushing or deletion. Overwatch users can now access historical audit logs for extended periods,
+facilitating in-depth analysis and compliance reporting.
+
+
+
+## System Table Integration Specific Differences
```
[the remark this answers is not in the document]
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md:54]: done.Please check

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md [hunk 15]
```diff
@@ -0,0 +1,76 @@
...
+or Azure Event Hubs.
+{{% /notice %}}
+The migration to system tables is straightforward. Organizations can effortlessly transition from existing 
+configurations to system table integrations, minimizing downtime and ensuring a smooth migration experience.
+Follow the below steps to migrate to System Table Integration: 
+[Configuration Details]({{%relref "DeployOverwatch/SystemTableIntegration/SystemTableConfiguration"%}})
+
+## Fetching data from Multi Account System Table  
+
+With the introduction of this new feature in Overwatch, we have expanded the capabilities to fetch system table data
```
[the remark this answers is not in the document]
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md:72]: done

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md (hunk 1 above)
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md:8]: done

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md (hunk 2 above)
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md:15]: done

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md (hunk 3 above)
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md:25]: done. change the statement to create new or use an existing warehouse

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md (hunk 4 above)
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md:24]: done

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTablePreReq.md (hunk 5 above)
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTablePreReq.md:6]: done

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTablePreReq.md (hunk 6 above)
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTablePreReq.md:11]: done

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTablePreReq.md (hunk 7 above)
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTablePreReq.md:17]: done

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTablePreReq.md (hunk 8 above)
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTablePreReq.md:18]: done

**Comment by aman-db** [comment]
> We also need to provide a new sample config .xlsx and .csv in "Configure Overwatch" -> "Configuration"

done

**Comment by aman-db** [comment]
> We need to add a note on the Configure Overwatch -> Validation page about validations performed for System tables

done

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (gueniai) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md:6]: The issue is that the integration IS supported from the Overwatch point of view. System tables are not available in GCP, which is a different story. If you want to keep the note then say that instead. 
IMO we should remove it

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (gueniai) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md:36]: I think we should still remove this, there's no real win with this point.

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md:6]: okay let me drop this

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+
+## Configuration changes required for System Table Integration
+
+There is no unique configuration process for System Table. The only change required to the configuration for integration
+is below - 
+
+* **auditlogprefix_source_path** - Instead of adding a fully qualified path (s3 or GC) for auditlog, 
+add keyword **system** in this column. This will enable the system table integration.
+
+**For migrating to system table from legacy above same congifuration changes can be used.**
```
Comment (gueniai) [inline docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md:14]: We need to remove this line. Legacy is no longer supported

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md:36]: Should we just merge these two points ?
* Simplified Setup within Overwatch
*Reduced Dependency on IAM Roles and Service Accounts

File: docs-compose/overwatch_docs/content/DeployOverwatch/ConfigureOverwatch/Configuration.md [hunk 17]
```diff
@@ -28,6 +28,10 @@ to you just leave it blank to avoid any strange issues when parsing the CSV.
 
 Please use the template for your version. If you are upgrading from 071x to 072x the upgrade script in the change log
 will upgrade the script for you.
+* 0.8.0.x+ ([.XLSX](/assets/DeployOverwatch/080x_overwatch_deployment_config_template.xlsx) |
+  [.CSV](/assets/DeployOverwatch/080x_overwatch_deployment_config_template.csv))
+  * Use this configuration for **multi account System Table Integration**
```
Comment (gueniai) [inline docs-compose/overwatch_docs/content/DeployOverwatch/ConfigureOverwatch/Configuration.md:33]: Let's remove this line, and have all users in 08x running with the latest config, regardless of multi-account

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md (hunk 16 above)
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md:14]: done

File: docs-compose/overwatch_docs/content/DeployOverwatch/ConfigureOverwatch/Configuration.md (hunk 17 above)
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/ConfigureOverwatch/Configuration.md:33]: done

File: docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md:6]: done. removed

**Review by gueniai — APPROVED** [review]
